test/test_browser.py

```diff
@@ -5541,10 +5541,20 @@ def test_error_reporting(self):
     create_file('post.js', 'throw "foo";')
     self.btest('hello_world.c', args=['--post-js=post.js'], expected='exception:foo')
 
-  def test_webpack(self):
-    shutil.copytree(test_file('webpack'), 'webpack')
+  @parameterized({
+    '': (False,),
+    'es6': (True,),
+  })
+  def test_webpack(self, es6):
+    if es6:
+      shutil.copytree(test_file('webpack_es6'), 'webpack')
+      self.emcc_args += ['-sEXPORT_ES6']
+      outfile = 'src/hello.mjs'
+    else:
+      shutil.copytree(test_file('webpack'), 'webpack')
+      outfile = 'src/hello.js'
     with utils.chdir('webpack'):
-      self.compile_btest('hello_world.c', ['-sEXIT_RUNTIME', '-sMODULARIZE', '-sENVIRONMENT=web', '-o', 'src/hello.js'])
+      self.compile_btest('hello_world.c', ['-sEXIT_RUNTIME', '-sMODULARIZE', '-sENVIRONMENT=web,worker', '-o', outfile])
```

Comment:
Why do we need `worker`? If webpack ends up running code in a worker in some magic way maybe add a comment about that?

Reply:
Ah, sorry, I tried and failed to get this test to run with pthreads, which required this.  Reverting for now.